### PR TITLE
@kanaabe => clean up file structure for edit, consolidate icons

### PR DESCRIPTION
**Pull request by eessex**
Consolidates the icons for the 'edit' app into one file.
Nests all section components under 'content' directory

screen shot 2017-06-14 at 5 44 40 pm

### CODE REVIEW

File: client/apps/edit/components/icons.jade [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+.callout
+  include ../public/icons/edit_menu_callout.svg
+.toc
+  include ../public/icons/edit_menu_toc.svg
+.image-set-menu
+  include ../public/icons/edit_menu_image_set.svg
+.link
+  include ../public/icons/edit_text_link.svg
+.image-set
+  include ../public/icons/edit_image_set_slideshow.svg
```
Comment (kanaabe) [inline client/apps/edit/components/icons.jade]: So much better!

**Comment by kanaabe** [comment]
Looks great! This could definitely use a second check on staging to make sure we didn't miss any paths -- thanks!

**Comment by kanaabe** [comment]
@eessex It looks like a few empty files were added to the root of the repo from this PR.`FETCH_HEAD` and `image-caption` for example.

**Comment by eessex** [comment]
hm, weird!  deleted these in my most recent pr - https://github.com/artsy/positron/pull/1129

**Pull request merged**